Project.toml

```
@@ -11,8 +11,8 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 StatsFuns = "4c63d2b9-4356-54db-8cca-17b64c39e42c" 
 
 [compat]
-AbstractGPs = "0.2"
-Distributions = "0.19, 0.20, 0.21, 0.22, 0.23"
+AbstractGPs = "0.2, 0.3"
+Distributions = "0.19, 0.20, 0.21, 0.22, 0.23, 0.24, 0.25"
 Functors = "0.1"
```

Can this be bumped as well?
```suggestion
Functors = "0.1, 0.2"
```